test/check_finufft.sh

```diff
@@ -27,6 +27,11 @@ else
     CHECK_TOL=1e-11
     export PRECSUF=
 fi
+if [[ $2 == "ON" ]]; then
```

Comment:
Adding this 2nd arg to check_finufft.sh seems neater than making a separate version of the script, so, good.  Could we ask you to document this 2nd arg in the # comments of the header for the script? Documentation of usage is essential. Thanks!

Reply:
I have added a few words on my changes, see: [ab7d95c](https://github.com/jonas-kr/finufft/commit/ab7d95c6d757f06a2c48d681db9a40b197684b38)

Reply:
It might make sense to put a .gitattributes file into the root directory which ensures that .sh files have LF-line endings on Windows.